tests/test_url_build.py

```diff
@@ -21,7 +21,7 @@ def test_url_build_ipv6():
 
 
 def test_url_build_ipv6_brackets():
-    u = URL.build(scheme="http", host="[::1]")
+    u = URL.build(scheme="http", host="[::1]", encoded=True)
```

Comment:
I'm a bit worried about having to change this test so I'm going to leave it for someone else to take a look before merging, otherwise I think its ready to go now.

Reply:
Once I fix https://github.com/aio-libs/yarl/pull/1158 if this still passes after re-basing, I'm now confident enough to merge this